project/CassandraSparkBuild.scala

```diff
@@ -41,20 +48,25 @@ object CassandraSparkBuild extends Build {
     ).copy(dependencies = Seq(embedded % "test->test;it->it,test;")
   ) configs (IntegrationTest, ClusterIntegrationTest)
 
-  lazy val jconnector = Project(
-    id = s"$namespace-java",
-    base = file(s"$namespace-java"),
-    settings = connector.settings,
-    dependencies = Seq(connector % "compile;runtime->runtime;test->test;it->it,test;provided->provided")
+  lazy val jconnector = CrossScalaVersionsProject(
+    name = s"$namespace-java",
+    conf = assembledSettings ++ Seq(libraryDependencies ++= Dependencies.connector ++ Seq(
+        "org.scala-lang" % "scala-reflect"  % scalaVersion.value,
+        "org.scala-lang" % "scala-compiler" % scalaVersion.value % "test,it"), compileOrder in Compile := CompileOrder.ScalaThenJava)
+    ).copy(dependencies = Seq(connector % "compile;runtime->runtime;test->test;it->it,test;provided->provided")
   ) configs (IntegrationTest, ClusterIntegrationTest)
 
   lazy val demos = RootProject("demos", demosPath, Seq(simpleDemos, kafkaStreaming, twitterStreaming))
 
-  lazy val simpleDemos = Project(
-    id = "simple-demos",
-    base = demosPath / "simple-demos",
-    settings = demoSettings,
-    dependencies = Seq(connector, jconnector, embedded)
+  lazy val simpleDemos =
+    Project(
```

Comment:
I'm not seeing the java api demo in this list anymore even when not using scala211